srt-protocol/src/listener/statistics.rs

```diff
@@ -0,0 +1,19 @@
+#[derive(Clone, Debug, Default, Eq, PartialEq)]
```

Comment:
Would be good to have these commented at some point, but not a huge deal for this PR

Reply:
Honestly, I was just throwing some things in there as a POC. I'm happy to remove them until it's better thought through.